scripts/science_db.lua

```diff
@@ -253,22 +253,39 @@ function __fillDefaultDatabaseData()
 	-- Populate the Factions top-level entry.
 	local faction_database = ScienceDatabase():setName(_("database", "Factions"))
 	for name, info in pairs(__faction_info) do
-        local entry = faction_database:addEntry(info.components.faction_info.locale_name);
+		local entry = faction_database:addEntry(info.components.faction_info.locale_name);
+
+		--[[ DB entries use locale-independent internal names.
+		     Sort the list of internal names by localized name to match the DatabaseView list.
+		     Lua in EE sorts by character index (for instance uppercase A > lowercase z), so lowercase the locale name for sorting purposes.
+		     Accents also break sorting, which for example affects the French translation of "Exuari" as "Éxuaris", which are listed last after MSU (USN).
+		     Since this isn't handled in DatabaseView on the C++ side either, this sorting is also "broken" her in the same manner for consistency.
```

Comment:
```suggestion
		     Since this isn't handled in DatabaseView on the C++ side either, this sorting is also "broken" here in the same manner for consistency.
```